requirements/requirements_asr.txt

```diff
@@ -20,3 +20,4 @@ pyannote.core
 pyannote.metrics
 pyamg
 torch-stft
+matplotlib
```

Comment:
This is unnecessary imo - we don't use it anywhere other than this script and colab and colab has it installed.

We can make a note in the code but making user install matplotlib just for an optional script is a bit heavy

Reply:
Actually this library is already in the requirements of TTS and NLP collections. It is a very common library and no harm to have it?

Reply:
Ok if TTS and NLP have it, might as well remove it from those 3 and push it up into the core requirements then. That cleanup can be done later if the domains agree. Looks good then.